public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -143,7 +144,7 @@ export function DataSourceModal({
 
   return (
     <Modal
-      title="Select data source"
+      title={t('data-source-picker.modal-title', 'Select data source')}
```

Comment:
```suggestion
      title={t('data-source-picker.modal.title', 'Select data source')}
```

Reply:
Thanks for reviewing,  I've made the changes.